modules/Microsoft.Compute/galleries/.test/images/deploy.test.bicep

```diff
@@ -1,89 +0,0 @@
-targetScope = 'subscription'
-
-// ========== //
-// Parameters //
-// ========== //
-@description('Optional. The name of the resource group to deploy for testing purposes.')
-@maxLength(90)
-param resourceGroupName string = 'ms.compute.galleries-${serviceShort}-rg'
-
-@description('Optional. The location to deploy resources to.')
-param location string = deployment().location
-
-@description('Optional. A short identifier for the kind of deployment. Should be kept short to not run into resource-name length-constraints.')
-param serviceShort string = 'cgimages'
-
-@description('Optional. Enable telemetry via a Globally Unique Identifier (GUID).')
-param enableDefaultTelemetry bool = true
-
-// =========== //
-// Deployments //
-// =========== //
-
-// General resources
-// =================
-resource resourceGroup 'Microsoft.Resources/resourceGroups@2021-04-01' = {
-  name: resourceGroupName
-  location: location
-}
-
-module resourceGroupResources 'dependencies.bicep' = {
-  scope: resourceGroup
-  name: '${uniqueString(deployment().name, location)}-paramNested'
-  params: {
-    managedIdentityName: 'dep-<<namePrefix>>-msi-${serviceShort}'
-  }
-}
-
-// ============== //
-// Test Execution //
-// ============== //
-
-module testDeployment '../../deploy.bicep' = {
```

Comment:
Was there a particular reason to remove the test? Were you just able to merge all into the `common` test and hence render this one irrelevant? :)

Reply:
I left common as is since It represents the commonly used parameters. I added the advanced because I thought these new settings wouldn't be used as much.

Reply:
Was merged into the common test